index.html

```diff
@@ -93,7 +92,7 @@
             </ul>
           </p>
       </section>
-      <section class="main-section" id="what_you_should_already_know">
+      <sction class="main-section" id="what_you_should_already_know">
```

Comment:
try to check the syntax of your html tags ,because using wrong syntax can lead to a wrong html segment which will cause alot of problem to the linters and seo.

Reply:
- [ ] Nice job mentioning the reason for the changes you asked but Kindly ask for the changes in a direct and more polite way and avoid the word Try to.